vault/dendron.tutorial.navigation-basics.md

```diff
@@ -2,47 +2,55 @@
 id: gHdxXlNMr1w4xqee0n-Mb
 title: Navigation Basics
 desc: 1. Navigation Basics
-updated: 1625573454967
+updated: 1639194721158
 created: 1625563862198
 nav_order: 0
 ---
 
 Let's do a brief overview on how to navigate the Dendron UI.
```

Comment:
rename the title since we have a functional area called navigation. 

eg. User Interface (https://code.visualstudio.com/docs/getstarted/userinterface#_explorer)